here-naksha-lib-hub/src/main/java/com/here/naksha/lib/hub/NakshaHubConfig.java

```diff
@@ -347,6 +354,23 @@ public static int defaultMaintenancePoolMaxSize() {
     return 5;
   }
 
+  /**
+   * Returns a default threshold per processor for concurrency
+   *
+   * @return the default threshold per processor
+   */
+  public static int defaultthresholdPerProcessor() {
```

Comment:
correct camel case = defaultThresholdPerProcessor()... anyways, the function needs renaming as per one of the previous comments

Reply:
Fixed it.